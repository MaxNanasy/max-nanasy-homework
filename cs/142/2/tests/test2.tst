void p1(){
a =1 ;
}
main(){
 a=1;
}