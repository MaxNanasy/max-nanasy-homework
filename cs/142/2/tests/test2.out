program
   procedureDeclarations
      retType
      statementSequence
         statement
            assignment
               designator
               expression
                  term
                     factor
   statementSequence
      statement
         assignment
            designator
            expression
               term
                  factor
